purchase_request_to_rfq/models/purchase_request.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# Copyright 2016 Acsone SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, exceptions, fields, models
+
+
+class PurchaseRequestLine(models.Model):
+
+    _inherit = "purchase.request.line"
+
+    @api.multi
+    @api.depends('purchase_lines')
+    def _compute_is_editable(self):
+        super(PurchaseRequestLine, self)._compute_is_editable()
+        for rec in self.filtered(lambda p: p in self and p.purchase_lines):
```

Comment:
Is it really required when filtering on `self` to check that the record is in `self` ?